protokube/pkg/protokube/aws_volume.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"k8s.io/klog/v2"
+
```

Comment:
I thought that running the gogumpt script would fix these issues...

Reply:
Not sure what caused this one